### ISSUE TITLE
Add `<app>`, `<rdg>`, `<lem>`

### ISSUE THREAD

**Issue by cthomasdta** [deutschestextarchiv/dtabf#69]
Wir brauchen früher oder später etwas zur Kodierung paralleler in einem Dokument. `<app>`, `<rdg>`, `<lem>` scheinen sich anzubieten. Hier nur erstmal als Merker, issue wird noch weiter ausgeführt und der Bedarf mit Anwendungsbeispiel plausibilisiert.

**Comment by wrznr** [comment]
> Wir brauchen früher oder später etwas zur Kodierung paralleler in einem Dokument

Paralleler was?

**Comment by cthomasdta** [comment]
> > Wir brauchen früher oder später etwas zur Kodierung paralleler in einem Dokument
> 
> Paralleler was?

Danke, ist ergänzt.

**Comment by xlhrld** [comment]
Das führt zunächst auf die grundsätzliche Frage, was im DTABf-Kontext ein »Text« sein soll. Vielleicht müssen wir uns zuerst damit beschäftigen?

**Comment by cthomasdta** [comment]
Nein, ich denke nicht. Wir haben konkrete Fälle vor der Tür, in denen diese Art Tagging verwendet wird, und für die die Hrsg. die Frage nach dem 'Text' somit schon beantwortet haben. M.E. gängige, gut etablierte und TEI-seitig gut unterstützte Praxis bei der Edition von Texten. Wenn das DTABf keine Möglichkeit enthält, dieses Tagging zu übernehmen, blieben nur:
1. die Texte nicht zu konvertieren und nicht ins DTA zu übernehmen
2. nur eine der in den jew. Dokumenten enthaltenen Textversionen zu übernehmen (= Verlust von Informationen)
3. die in einem Dokument zugleich kodierten mehreren Dokumente wieder aufzusplitten -- was im Ergebnis als Prüfstein des Taggings interessant sein kann, aber sehr aufwändig wäre.

**Comment by burki** [comment]
@cthomasdta Kannst du den Anwendungsfall noch etwas spezifizieren? Geht es um eine gedruckte Edition mit Apparat, die nun in TEI ausgezeichnet wird, oder geht es darum, einen historischen Text (ohne Apparat) in der digitalen Fassnung mit  einem kritischen Apparat zu ergänzen.

**Comment by cthomasdta** [comment]
Cf. https://bdn-edition.de/modeledition.html?file=2501b.2.xml&edition=foreword&model=true

z.B. (etwas vereinfachtes Beispiel):
`                                    <app>
                                        <lem wit="#a">Auch bey</lem>
                                        <rdg wit="#b">Bey</rdg>
                                        <rdg wit="#c">Bei</rdg>
                                    </app> der alten Erdbeschreibung`

Es geht also darum, verschiedene (historische) Auflagen eines Textes in einer TEI-XML-Datei zu kodieren. Auf deren Beschreibung im teiHeader wird mittels `#a`, `#b`, `#c` verwiesen.

Macht es das verständlicher?

**Comment by StefanDumont** [comment]
Wir haben für ediarum bereits einen Variantenapparat in TEI-XML entwickelt, der in Kürze implementiert wird. An der Entwicklung war auch Susanne beteiligt, das sollten wir daher als Ausgangspunkt nehmen oder gleich ganz übernehmen, falls wir uns grundsätzlich dafür entscheiden.

**Comment by cthomasdta** [comment]
Na klar! Dann können wir diesen Vorschlag diskutieren, ich bin überhaupt nicht auf das o.g. "Modell" festgelegt. Bitte teilt dann diesen Vorschlag hier mit, wenn ihr so weit seid. Danke!

**Comment by textloop** [comment]
Für den Pilot-Band der Leibniz-Edition in TEI habe ich es aktuell (auch in Rücksprache mit Telota) ähnlich eingesetzt, die Abnahme des Endergebnisses steht allerdings noch aus: 
```
<app type="Bfootnote">
<lem type="main"><hi rendition="#i">cum ex diametro una</hi></lem>
...
<rdg wit="#L" n="1"> diametris duabus </rdg>
<rdg wit="#L" n="2"> <hi rendition="#i">diametro una</hi> </rdg>
</app>
```
Der Haupttext zeigt die editorisch erarbeitete finale Druckfassung und wird im teiHeader als solcher ausgewiesen. Deswegen unterschlage ich hier im `<lem>` das @wit. Die einzelnen Fassungen in `<reg>` tragen jeweils ein @wit, hier "#L" (Leibniz). Da können auch mal andere Wert erscheinen. 


[Inhaltliche Erläuterung:

Wir modellieren hier eine gedruckte Edition mit Anmerkungsapparaten, nicht die Handschriftenvorlage selbst. 
Der Eintrag zeigt einen Stufenapparateintrag:
Erst stand in der Leibniz-Handschrift: "diametris duabus"[rdg1], dann hat er korrigiert zu "diametro una"[rdg2]. Die im Haupttext der Edition abgedruckte finale Fassung lautet: "diametro una"[lem].]

**Comment by cthomasdta** [comment]
NB Tool interessant dafür: https://github.com/HuygensING/hyper-collate

**Comment by susannehaaf** [comment]
Ich meine, der Variantenapparat, den wir seinerzeit für ediarum diskutiert hatten, sah folgendes vor (bitte @StefanDumont korrigiere/ergänze, falls nötig):

* im `<text>`:
  * Element `<app>` mit `@type="variants"`, in `<app>`können dann stehen:
  * Element `<rdg>` mit `@wit`
  * Element `<lem>` mit `@wit` und `@cause`
  * für `@cause`die Werte: 
    * `orthographic` - Abweichung durch Schreibfehler
    * `correction` - Abweichung durch Korrektur
    * `conjecture` - Vermutetes Lemma ohne Quellennachweis
    * `margin` - Lesart steht am Rand
    * `omission` - Auslassung
    * `addition` - Hinzufügung 
    * `transposition` - Ersetzung
    * `supralinear` - Hinzufügung zwischen den Linien
    * `homoioteleuton` - Auslassung aufgrund gleicher Wortendung
    * `homoioarkton` - Auslassung aufgrund gleicher Wortanfänge
    * `erasion` - Auslassung aufgrund einer Löschung 
  * ggf. `<note type="editorial">`
* innerhalb von `<sourceDesc>` im `teiHeader`:
  * Element `<witness>` mit `@xml:id`, um die Textzeugen zu beschreiben
  * Element `<listWit>` im für gruppierte Textzeugen
  * ggf. weitere `@types` für `<idno>` innerhalb von `<witness>`
* evtl. innerhalb einer Variante die Option, textkritische Elemente zu verwenden: `<rdg wit="h1">ist ein <add>neuer</add> Satz</rdg>`
* offen blieb: Absatzwechsel (oder ein anderer Wechsel der Textstruktur) innerhalb einer "Lesart" oder Variante sowie Wechsel von Textstrukturen in einer Variante im Gegensatz zu anderen Varianten, wo dieser Wechsel nicht steht: Vorschlag war, mit milestone-Elementen (`<lem> Text <milestone type="p"/> Neue Zeile</lem>`) oder mit `<anchor>` sowie `@from/@to` zu arbeiten)

**Comment by susannehaaf** [comment]
Beschluss der DTABf-Steuerungsgruppe vom 7.1.2021:

- Nutzung von `<lem>`, `<rdg>`, `<app>` sowie der entsprechend nötigen TEI-Header-Elemente wird anhand eines DTA-Kooperationsprojekts geprüft (i.e. Bibliothek der Neologie; Link s. oben)
- `@cause` wird zunächst nicht eingeplant

**Comment by StefanDumont** [comment]
> Nutzung von <lem>, <rdg>, <app> sowie der entsprechend nötigen TEI-Header-Elemente wird anhand eines DTA-Kooperationsprojekts geprüft (i.e. Bibliothek der Neologie; Link s. oben)

Gibt es hierzu irgendwie Feedback o.ä. mittlerweile, das wir für die weitere Behandlung nutzen können?

**Comment by cthomasdta** [comment]
Gute Frage! Und bitte zur Erinnerung:

> `@cause` wird zunächst nicht eingeplant

Was sprach gegen `@cause` ? 

Propyläen werden es einsetzen, zB `<subst cause="undo">` bei Zurücknahme einer Streichung (zB durch Unterpungierung in der Handschrift)
